qa/L0_sdk/test.sh

```diff
@@ -161,6 +161,21 @@ for l in $WHLS; do
     fi
 done
 
+# Check wheel installation
+python -c """import tritonclient; import tritonclient.grpc; import tritonclient.http; \
+          import tritonclient.utils; import tritonclient.grpc.model_config_pb2; \
+          import tritonclient.grpc.service_pb2; import tritonclient.grpc.service_pb2_grpc; \
+          import tritonclient.utils.cuda_shared_memory; import tritonclient.utils.shared_memory"""
+RET_VAL=$(($RET_VAL+$?))
+EXECUTABLES="perf_analyzer perf_client"
+for l in $EXECUTABLES; do
+  if [ $(which -a $l | grep "/usr/local/bin/$l" | wc -l) -ne 1 ]; then
+    which -a $l
```

Comment:
Is this line need?

Reply:
In case of failure we will print all the locations of executables in the system path. This might help us in understanding what is wrong right away.